### ISSUE TITLE
Neufund's % of limit 

### ISSUE THREAD

**Issue by cgewecke** [cgewecke/eth-gas-reporter#9]
They have a nice gas logging utility in their tests that looks like this:

![screen shot 2017-10-02 at 7 41 48 pm](https://user-images.githubusercontent.com/7332026/31107786-ce740820-a7a9-11e7-86ed-d8fea49c4500.png)

387% of the limit. The source data for the gasPrice and ether price is static.

**Comment by cgewecke** [comment]
Their math: from [here](https://github.com/Neufund/ico-contracts/blob/master/test/helpers/gasUtils.js#L7-L10)
```javascript
const gasCostString = gas =>
  `${gas} gas (€${Math.round(100 * gas * gasPrice) / 100}, ${Math.round(
    1000 * gas / gasLimit
  ) / 10}% of limit)`;
```

**Comment by cgewecke** [comment]
Published w/ `0.0.5`
